sklearn/model_selection/tests/test_successive_halving.py

```diff
@@ -562,3 +569,40 @@ def set_params(self, **params):
 
     assert (cv_results_df['params'] == passed_params).all()
     assert (cv_results_df['n_resources'] == passed_n_samples).all()
+
+
+def test_groups_support():
```

Comment:
Would you mind parametrizing this one with both HalvingSearchCV and RandomizedHalvingSearchCV, as done in the other test functions of this file? thanks!

e.g. `@pytest.mark.parametrize('Est', (HalvingGridSearchCV, HalvingRandomSearchCV))`

Reply:
Good call @NicolasHug . Changed in the latest commit. Thanks.